templates/login.twig

```diff
@@ -16,16 +21,28 @@
 				{% endif %} 
 			{% endif %} 
 		{% endif %}
-		<form method="post" name="loginForm"
+		<div class="row">
+		<div class="col-8">
+		<h2>What is squareBracket?</h2>
+		squareBracket is a way to get your videos to the people who matter to you. With squareBracket you can:
+		<ul>
+		<li> Show off your favorite videos to the world
+		</li><li> Blog the videos you take with your digital camera or cell phone
+		</li><li> Securely and privately show videos to your friends and family around the world
+		</li><li> ... and much, much more!
```

Comment:
this will need to be rewritten, it's stolen text from a 16 year old copy of youtube.